.github/workflows/main_cd.yml

```
@@ -3,27 +3,43 @@ name: Create a release on NuGet
 on:
   push:
     tags:
-    - "v[0-9]+.[0-9]+.[0-9]+"
-    - "v[0-9]+.[0-9]+.[0-9]+-pre[0-9]+"
+      - "v[0-9]+\\.[0-9]+\\.[0-9]+"
```

actionlint (1.7.4)

character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)